assignments/prototypes.js

```diff
@@ -32,6 +58,15 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
+  function  Humanoid(attrs) {
```

Comment:
The Humanoid class has missing attributes and functions. Those are specs for the class:

```
* team
  * weapons
  * language
  * greet() // prototype method -> returns the string '<object name> offers a greeting in <object language>.'
  * should inherit destroy() from GameObject through CharacterStats
  * should inherit takeDamage() from CharacterStats
```

It should also inherit from `CharacterStats` Object.